src/js/components/Carousel/doc.js

```diff
@@ -27,6 +27,9 @@ export const doc = Carousel => {
       will loop through all children indefinitely.`),
     initialChild: PropTypes.number.description(`If specified, the index of
       the first element to be shown. Defaults to 0.`),
+    onChild: PropTypes.func.description(`If specified, this function will 
+      be called with the active index when the currently active carousel 
+      changes.`),
```

Comment:
Not sure this message makes sense to me. The users are not aware of what active index is, so maybe it will be more understood to avoid using it.

Reply:
Hi @ShimiSun,

I took this explanation from grommet's doc v1 because that's exactly what this function does in this PR. (https://v1.grommet.io/docs/carousel)

![image](https://user-images.githubusercontent.com/36260986/104289601-9f1b9500-54b9-11eb-99ce-26f31fc76f30.png)

I think people who use React know what an active index is.
And as far as the word index is concerned, it is used in the documentation of grommet v2.

![image](https://user-images.githubusercontent.com/36260986/104290045-20732780-54ba-11eb-91bf-083d4db6fe9e.png)

That being said, I understood what you meant and I offer you this explanation: 

_`If specified, this function will be called with the index of the element shown when the currently active carousel changes.`_

What do you think?